pkg/es/config/config.go

```diff
@@ -476,6 +476,7 @@ func addLoggerOptions(options []elastic.ClientOptionFunc, logLevel string) ([]el
 
 		// Enables the "level":"debug" log field. Without this,
 		// the "level" field defaults to "info".
+		// TODO: what does grpc have to do with setting the ES logger?
```

Comment:
@albertteoh do you remember why zapgrpc is being called here at all? First commit https://github.com/jaegertracing/jaeger/pull/2862/files#diff-de5aae4da3cb03d3a73919aca1f6f38181b906dfd205bb0def4bf7b9d6042992R408

Why wouldn't `prodConfig.Level.SetLevel(lvl)` in L491 achieve the same thing?

`zapgrpc.WithDebug` is deprecated, and using the recommended `zapgrpc.SetLoggerV2` seems like going in completely wrong direction.